gatsby-node.js

```diff
@@ -219,11 +248,21 @@ let createDatumDescription = async (ctd, datum, foreignReferenceMap) => {
 
 const createTypeDefs = (contentTypesDefinitions) => {
     let typeDefs = '';
+    let additionalDefs = {};
     contentTypesDefinitions.forEach(ctd => {
         let tmpDef = '';
         Object.keys(ctd.schemaDefinition.allOf[1].properties).forEach(property => {
             tmpDef = tmpDef + `
-            ` + property + `: ` + getType(ctd.metaDefinition.propertiesConfig[property], ctd.schemaDefinition.required.indexOf(property) > -1)
+            ` + property + `: ` + getType(ctd.metaDefinition.propertiesConfig[property], ctd.schemaDefinition.required.indexOf(property) > -1, property, capitalize(ctd.name))
+            if(ctd.metaDefinition.propertiesConfig[property].inputType === 'object') {
+                additionalDefs[capitalize(property) + capitalize(ctd.name)] = '';
+                Object.keys(ctd.metaDefinition.propertiesConfig[property].items.propertiesConfig).forEach(prop => {
+                    additionalDefs[capitalize(property) + capitalize(ctd.name)] =  additionalDefs[capitalize(property) + capitalize(ctd.name)] + `
+                  ` + prop + `: ` + getType(ctd.metaDefinition.propertiesConfig[property].items.propertiesConfig[prop], false, prop, capitalize(ctd.name))
+                });
+                additionalDefs[capitalize(property) + capitalize(ctd.name)] = additionalDefs[capitalize(property) + capitalize(ctd.name)] + `
+                        flotiqInternal: FlotiqInternal!`;
```

Comment:
It might be a bit unreadable, so the below suggestion does 2 things:
- extracts `definitionId` constant, which serves as an index in `additionalDefs`
- uses Template literals (Template strings) to build a definition instead of concatenation
```suggestion
                const definitionId = capitalize(property) + capitalize(ctd.name);
                additionalDefs[definitionId] = '';
                Object.keys(ctd.metaDefinition.propertiesConfig[property].items.propertiesConfig).forEach(prop => {
                    additionalDefs[definitionId] =  `${additionalDefs[definitionId]}
                  ${prop}: ${getType(ctd.metaDefinition.propertiesConfig[property].items.propertiesConfig[prop], false, prop, capitalize(ctd.name))}`
                });
                additionalDefs[definitionId] = `${additionalDefs[definitionId]}
                        flotiqInternal: FlotiqInternal!`;
```